lib/Drush/Commands/core/SanitizeCommands.php

```diff
@@ -0,0 +1,274 @@
+<?php
+
+namespace Drush\Commands\core;
+
+use Drupal\Component\Utility\Random;
+use Drupal\Core\Database\Database;
+use Drush\Commands\DrushCommands;
+
+/**
+ * Class SanitizeCommands
+ * @package Drush\Commands\core
+ */
+class SanitizeCommands extends DrushCommands {
+
+  /**
+   * @var bool
+   *   Whether database table names should be wrapped in brackets for prefixing.
+   */
+  protected $wrap;
+
+  /**
+   * Sets $this->wrap to TRUE if a db-prefix is set with drush.
+   */
+  protected function setWrap() {
+    $this->wrap = $wrap_table_name = (bool) drush_get_option('db-prefix');
+  }
+
+
+  /**
+   * Initialize local Drush configuration
+   *
+   * @command sql-sanitize
+   *
+   * @todo "drush dependencies" array('sqlsync')
+   *
+   * @bootstrap DRUSH_BOOTSTRAP_NONE
+   * @description Run sanitization operations on the current database.
+   * @option db-prefix Enable replacement of braces in sanitize queries.
+   * @option db-url A Drupal 6 style database URL. E.g.,
+   *   mysql://root:pass@127.0.0.1/db
+   * @option sanitize-email The pattern for test email addresses in the
+   *   sanitization operation, or "no" to keep email addresses unchanged. May
+   *   contain replacement patterns %uid, %mail or %name. Example value:
+   *   user+%uid@localhost
+   * @option sanitize-password The password to assign to all accounts in the
+   *   sanitization operation, or "no" to keep passwords unchanged. Example
+   *   value: password
+   * @option whitelist-fields Exempt these fields from sanitizing.
+   * @aliases sqlsan
+   * @usage drush sql-sanitize --sanitize-password=no
+   *   Sanitize database without modifying any passwords.
+   * @usage drush sql-sanitize --whitelist-fields=field_biography,field_phone_number
+   *   Sanitizes database but exempts two user fields from modification.
+   * @see hook_drush_sql_sync_sanitize() for adding custom sanitize routines.
+   */
+  public function sqlSanitize($options = [
+    'db-prefix' => '',
```

Comment:
Is --db-prefix a string or a boolean. Lets clarify via help text and/or default value in params.